app/views/course/assessment/question/text_responses/_form.html.slim

```diff
@@ -1,38 +0,0 @@
-= simple_form_for [current_course, @assessment, @text_response_question] do |f|
-  = f.error_notification
-  = render partial: 'course/assessment/questions/form', locals: { f: f, question_assessment: @question_assessment }
```

Comment:
Remove this file if no longer used

Reply:
Have u deleted the file in this line?

Reply:
this one is needed for rendering the common field among all the question forms (title, description, maximum grade, staff only comment, and skill Id)

Reply:
Are you sure? Is it still used elsewhere?

Reply:
yes.. it's used across all the question forms. btw, I think you meant that `course/assessment/question/form` to be `app/views/course/assessment/question/_form.json.jbuilder`, right?

Reply:
is it json or html slim?

Reply:
the one referred by rails shall be in html, but that one has been deleted long ago.. now what's inside `course/assessment/question`, which is not a dictionary, is only `_form.json.jbuilder` and `_skills.json.jbuilder`, and nothing else

Reply:
oh my bad.. I'm sorry I misread that questions as question. now I'm deleting it